lib/matplotlib/backends/qt_editor/_formlayout.py

```diff
@@ -537,8 +536,7 @@ def fedit(data, title="", comment="", icon=None, parent=None, apply=None):
             parent._fedit_dialog.close()
         parent._fedit_dialog = dialog
 
-    if qt_compat._exec(dialog):
-        return dialog.get()
+    dialog.show()
```

Comment:
This change lead to that `fedit` does not return any answer (other than `None`) leading to that the documentation and probably the examples where the result is printed should be changed.